pkg/exchange/bitget/exchange.go

```diff
@@ -397,18 +397,22 @@ func (e *Exchange) QueryOpenOrders(ctx context.Context, symbol string) (orders [
 // QueryClosedOrders queries closed order by time range(`CreatedTime`) and id. The order of the response is in descending order.
 // If you need to retrieve all data, please utilize the function pkg/exchange/batch.ClosedOrderBatchQuery.
 //
+// REMARK: If your start time is 90 days earlier, we will update it to now - 90 days.
 // ** Since is inclusive, Until is exclusive. If you use a time range to query, you must provide both a start time and an end time. **
 // ** Since and Until cannot exceed 90 days. **
-// ** Since from the last 90 days can be queried. **
+// ** Since from the last 90 days can be queried **
 func (e *Exchange) QueryClosedOrders(ctx context.Context, symbol string, since, until time.Time, lastOrderID uint64) (orders []types.Order, err error) {
-	if time.Since(since) > queryMaxDuration {
-		return nil, fmt.Errorf("start time from the last 90 days can be queried, got: %s", since)
+	newSince := since
+
+	if time.Since(newSince) > queryMaxDuration {
```

Comment:
name it maxHistoricalDataQueryPeriod?

Reply:
and you should also modify the "until"?

Reply:
Are you suggesting changing 'until' to 'now'? I did consider it, but later on, I noticed that everyone else was using 'time.Now,' so I didn't make the change